ext/opcache/zend_accelerator_module.c

```diff
@@ -700,16 +700,11 @@ ZEND_FUNCTION(opcache_get_status)
 		}
 
 		if (zend_hash_num_elements(&ZCSG(preload_script)->script.class_table)) {
-			zend_class_entry *ce;
 			zend_string *key;
 
 			array_init(&scripts);
-			ZEND_HASH_MAP_FOREACH_STR_KEY_PTR(&ZCSG(preload_script)->script.class_table, key, ce) {
-				if (ce->refcount > 1 && !zend_string_equals_ci(key, ce->name)) {
-					add_next_index_str(&scripts, key);
-				} else {
-					add_next_index_str(&scripts, ce->name);
-				}
+			ZEND_HASH_MAP_FOREACH_STR_KEY(&ZCSG(preload_script)->script.class_table, key) {
+				add_next_index_str(&scripts, key);
```

Comment:
This changes the behavior.

Reply:
The difference in behaviour is that all names will be lowercase now, I thought that was okay.
I'll  change it back to use a case insensitive equality check